hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMStateMachine.java

```diff
@@ -80,6 +81,8 @@
   // and reinitialize().
   private DBCheckpoint installingDBCheckpoint = null;
 
+  private Daemon leaderReady;
```

Comment:
leaderReady -> leaderReadyDetector

Reply:
Please add some doc on what this thread is supposed to do.

Reply:
Removed this